docs/bin/plugin_formatter.py

```diff
@@ -32,6 +32,8 @@
 from collections import defaultdict
 from distutils.version import LooseVersion
 from pprint import PrettyPrinter
+from functools import partial
+from ansible.module_utils.common.collections import is_sequence
```

Comment:
Could you please move this next to line 53? We tend to logically group imports.

Reply:
Sure, but then you already did that yourself ;-)